src/components/CodeInput/styled/index.js

```diff
@@ -83,6 +89,11 @@ const StyledInput = attachThemeAttrs(styled.input)`
     `}
 `;
 
+const StyledInputItems = attachThemeAttrs(styled.div)`
+    position: relative;
+    margin-bottom: -12px;
```

Comment:
@TahimiLeonBravo check this negative margin

Reply:
![Screen Shot 2020-05-14 at 16 56 14](https://user-images.githubusercontent.com/60666952/81990010-eb00fc00-9603-11ea-80ec-f9a8a2f01d4b.png)

that was because in a previous commit I added to inputs a 12px margin-bottom to prepare for responsiveness,  so I added to the container a negative margin bottom to bring back moved the position.

![Screen Shot 2020-05-14 at 16 57 47](https://user-images.githubusercontent.com/60666952/81990120-24d20280-9604-11ea-8daa-689016f5eb1d.png)

Reply:
@blakmetall @LeandroTorresSicilia I think we should not add this negative margin since this a bad practice if we want to separate the bottomHelpText or the errorText from the input we should remove the `margin-top`